### ISSUE TITLE
Add support for `iOS Simulators` with paired mac

### ISSUE THREAD

**Issue by richardst89** [JaneySprings/DotNet.Meteor#144]
Following on from this guide, it'd be great to be able to utilise a simulator for iOS development, rather than a fully fledged machine.

They are installed as part of the dotnet workload, so assuming this is feasible!!

https://learn.microsoft.com/en-us/dotnet/maui/ios/remote-simulator?view=net-maui-9.0

**Comment by JaneySprings** [comment]
Yes, I think that it's possible, but I don't have any documentation about it. For iOS devices on Windows, Microsoft uses this open-source library: https://github.com/libimobiledevice/libimobiledevice

We need to explore the dotnet workloads for it.
